docs/webactions.md

```diff
@@ -286,29 +287,79 @@ Notice in this case the JSON content is base64 encoded because it is treated as
 
 ### Enabling raw HTTP handling
 
-Raw HTTP web actions are enabled via the annotation `raw-http` [annotation](annotations.md) with a value of `true`.
+Raw HTTP web actions are enabled via the `--web` flag using a value of `raw`.
 
 ```bash
-$ wsk action create /guest/demo/hello hello.js \
-      --annotation web-export true
-      --annotation raw-http true
+$ wsk action create /guest/demo/hello hello.js --web raw
 ```
 
-**Note:** Since `raw-http` implies `web-export`, we plan to improve the CLI to provide a more convenient way to add (and remove) these annotations in the future.
-
-
 ### Disabling raw HTTP handling
 
-Disabling raw HTTP is accomplished by setting the `raw-http` [annotation](annotations.md) value to `false`.
+Disabling raw HTTP can be accomplished by passing a value of `false` or `no` to the `--web` flag.
 
 ```bash
-$ wsk update create /guest/demo/hello hello.js \
-      --annotation web-export true
-      --annotation raw-http false
+$ wsk update create /guest/demo/hello hello.js --web false
+```
+
+### Decoding binary body content from Base64
+
+When using raw HTTP handling, the `__ow_body` content will be encoded in Base64 when the request content-type is binary.
+Below are functions demonstrating how to decode the body content in Node, Python, and Swift. Simply save a method shown
+below to file, create a raw HTTP web action utilizing the saved artifact, and invoke the web action.
+
+#### Node
+
+```javascript
+function main(args) {
+    decoded = new Buffer(args.__ow_body, 'base64').toString('utf-8')
+    return {body: decoded}
+}
+```
+
+#### Python
+
+```python
+def main(args):
+    try:
+        decoded = args['__ow_body'].decode('base64').strip()
+        return {"body": decoded}
+    except:
+        return {"body": "Could not decode body from Base64."}
 ```
 
-**Note:**  All annotations for a single action must be set at the same time, either when creating or updating the action. This is due to a current limitation on the API and CLI. Failure to do so will result is removal of any previously attached annotations.
+#### Swift
+
+```swift
+extension String {
+    func base64Decode() -> String? {
+        guard let data = Data(base64Encoded: self) else {
+            return nil
+        }
+
+        return String(data: data, encoding: .utf8)
+    }
+}
+
+func main(args: [String:Any]) -> [String:Any] {
+    if let body = args["__ow_body"] as? String {
+        if let decoded = body.base64Decode() {
+            return [ "body" : decoded ]
+        }
+    }
+
+    return ["body": "Could not decode body from Base64."]
+}
+```
 
+As an example, save the NodeJS function as `decode.js` and execute the following commands:
```

Comment:
It looks like in their docs they refer to "Node" and other official places as "Node.js" if I have to pick then "Node"